apps/DemoApp/run-tests.sh

```diff
@@ -6,7 +6,13 @@ rm -Rf XCFramework
 # reset simulators
 xcrun -v simctl shutdown all
 xcrun -v simctl erase all
-buildkite-agent artifact download "MUXSDKStats.xcframework.zip" . --step ".buildkite/build.sh"
+
+# Fetch artifact if running via buildkite (GitHub Actions fetches the artifact in a prior step)
+if command -v buildkit-agent > /dev/null 2>&1;
```

Comment:
oh nice, so we can support buildkite still if needed. Thanks for adding this

Reply:
Oops! It looks like I may have gotten something wrong with my bash conditional here, based on this recent buildkite failure:

> `unzip:  cannot find or open MUXSDKStats.xcframework.zip, MUXSDKStats.xcframework.zip.zip or MUXSDKStats.xcframework.zip.ZIP`

https://buildkite.com/mux/stats-sdk-avplayer/builds/483#7f30c47a-b87b-48cd-8cb2-d7611b451029

Reply:
Oh, I see it now. `s/buildkit/buildkite/` 🤦‍♂️

Reply:
Ah good catch. I actually just deactivated the webhook for the Buildkite pipeline but can reactivate it if you want to test

Reply:
I did open [a PR](https://github.com/muxinc/mux-stats-sdk-avplayer/pull/140) to fix, but I think this really only affects my pride. 😆